sms_sponsorship/__init__.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2018 Compassion CH (http://www.compassion.ch)
+#    @author: Quentin Gigon <gigon.quentin@gmail.com>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+from . import models
```

Comment:
You should put a init file in the models directory as well, but with no import inside